SimPEG/Maps.py

```diff
@@ -347,6 +357,161 @@ def deriv(self, m, v=None):
         return self.P
 
 
+class SumMap(ComboMap):
```

Comment:
@fourndo: can you take a look at this? This should do the same as is in pr #666 (which has to be bad luck if we merge it!, ha)